website/src/data/events-listing.json

```
@@ -1,25 +1,26 @@
 {
-"Conference":[
-	{
-		"eventTitle":"TSConf JP 2020",
-		"eventDate": "February 22, 2020",
-		"eventLocation":"Tokyo, Japan",
-		"eventLink": "https://www.tsconf.jp/2020/",
-		"eventImage": "/external-logos/tsconf-js-logo.svg"
-	},
-	{
-		"eventTitle":"RxJS Live",
-		"eventDate": "March 20, 2020",
-		"eventLocation":"London, England",
-		"eventLink": "https://www.rxjs.live/",
-		"eventImage": "/external-logos/rxjs-live-london-logo.svg"
-	},
-	{
-		"eventTitle":"TSConf EU 2020",
-		"eventDate": "March 31, 2020",
-		"eventLocation":"Linz, Austria",
-		"eventLink": "https://tsconf.eu/",
-		"eventImage": "/external-logos/tsconf-eu-logo.svg"
-	}	
-]
-}
+	"Conference": [
+		{
+			"eventTitle":"RxJS Live",
+			"eventDate": "March 20, 2020",
+			"eventLocation":"London, England",
+			"eventLink": "https://www.rxjs.live/",
+			"eventImage": "/external-logos/rxjs-live-london-logo.svg"
+		},
+		{
+			"eventTitle":"TSConf EU 2020",
+			"eventDate": "March 31, 2020",
+			"eventLocation":"Linz, Austria",
+			"eventLink": "https://tsconf.eu/",
+			"eventImage": "/external-logos/tsconf-eu-logo.svg"
+		}
```

```suggestion
		}
```

TSConf EU has been cancelled.

---

😲

---

Then we need to update the `col-sm-` class accordingly.